test/app/app.rb

```diff
@@ -26,7 +26,7 @@
 
 org = Organization.create!(name: 'test')
 org.users.create!(email: 'squirrel@example.com')
-users = User.where(email: 'squirrel@example.com').to_a # bumpity
+users = User.where(email: 'squirrel@example.com').to_a # bumpity-bump
```

Comment:
* Table Scan: postgres reads 100% (100000) of the of the rows in **users**, skipping any indexes.
 * Estimated query time: 0.02s